CI failing with tons of dead dotty.epfl.ch links

see e.g. https://github.com/scala/scala-lang/runs/5193204739 

I'm not exactly sure whose attention to call this to. did something get merged recently in the dotty repo that would cause this?

```
11
- ./_site/2019/01/18/community-build.html
[12](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:12)
  *  External link [dotty.epfl.ch/blog/2018/11/30/11th-dotty-milestone-release.html](https://dotty.epfl.ch/blog/2018/11/30/11th-dotty-milestone-release.html) failed: 404 No error
[13](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:13)
- ./_site/2019/12/18/road-to-scala-3.html
[14](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:14)
  *  External link [dotty.epfl.ch/docs/reference/dropped-features/delayed-init.html](https://dotty.epfl.ch/docs/reference/dropped-features/delayed-init.html) failed: 404 No error
[15](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:15)
  *  External link [dotty.epfl.ch/docs/reference/dropped-features/early-initializers.html](https://dotty.epfl.ch/docs/reference/dropped-features/early-initializers.html) failed: 404 No error
[16](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:16)
  *  External link [dotty.epfl.ch/docs/reference/dropped-features/existential-types.html](https://dotty.epfl.ch/docs/reference/dropped-features/existential-types.html) failed: 404 No error
[17](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:17)
  *  External link [dotty.epfl.ch/docs/reference/dropped-features/weak-conformance.html](https://dotty.epfl.ch/docs/reference/dropped-features/weak-conformance.html) failed: 404 No error
[18](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:18)
  *  External link [dotty.epfl.ch/docs/reference/features-classification.html#dropped-constructs](https://dotty.epfl.ch/docs/reference/features-classification.html#dropped-constructs) failed: 404 No error
[19](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:19)
  *  External link [dotty.epfl.ch/docs/reference/metaprogramming/inline.html](https://dotty.epfl.ch/docs/reference/metaprogramming/inline.html) failed: 404 No error
[20](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:20)
  *  External link [dotty.epfl.ch/docs/reference/other-new-features/control-syntax.html](https://dotty.epfl.ch/docs/reference/other-new-features/control-syntax.html) failed: 404 No error
[21](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:21)
  *  External link [dotty.epfl.ch/docs/reference/other-new-features/indentation.html](https://dotty.epfl.ch/docs/reference/other-new-features/indentation.html) failed: 404 No error
[22](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:22)
  *  External link [dotty.epfl.ch/docs/usage/ide-support.html](https://dotty.epfl.ch/docs/usage/ide-support.html) failed: 404 No error
[23](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:23)
- ./_site/2020/11/06/explicit-term-inference-in-scala-3.html
[24](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:24)
  *  External link [dotty.epfl.ch/docs/reference/contextual/context-functions.html](https://dotty.epfl.ch/docs/reference/contextual/context-functions.html) failed: 404 No error
[25](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:25)
  *  External link [dotty.epfl.ch/docs/reference/contextual/conversions.html](https://dotty.epfl.ch/docs/reference/contextual/conversions.html) failed: 404 No error
[26](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:26)
  *  External link [dotty.epfl.ch/docs/reference/contextual/extension-methods.html](https://dotty.epfl.ch/docs/reference/contextual/extension-methods.html) failed: 404 No error
[27](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:27)
  *  External link [dotty.epfl.ch/docs/reference/contextual/type-classes.html](https://dotty.epfl.ch/docs/reference/contextual/type-classes.html) failed: 404 No error
[28](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:28)
  *  External link [dotty.epfl.ch/docs/reference/other-new-features/opaques.html](https://dotty.epfl.ch/docs/reference/other-new-features/opaques.html) failed: 404 No error
[29](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:29)
- ./_site/2021/02/26/tuples-bring-generic-programming-to-scala-3.html
[30](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:30)
  *  External link http://dotty.epfl.ch/docs/reference/contextual/extension-methods.html failed: 404 No error
[31](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:31)
  *  External link http://dotty.epfl.ch/docs/reference/new-types/dependent-function-types.html failed: 404 No error
[32](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:32)
  *  External link http://dotty.epfl.ch/docs/reference/new-types/match-types.html failed: 404 No error
[33](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:33)
  *  External link [dotty.epfl.ch/docs/reference/contextual/context-bounds.html](https://dotty.epfl.ch/docs/reference/contextual/context-bounds.html) failed: 404 No error
[34](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:34)
  *  External link [dotty.epfl.ch/docs/reference/contextual/derivation.html](https://dotty.epfl.ch/docs/reference/contextual/derivation.html) failed: 404 No error
[35](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:35)
  *  External link [dotty.epfl.ch/docs/reference/contextual/derivation.html#types-supporting-derives-clauses](https://dotty.epfl.ch/docs/reference/contextual/derivation.html#types-supporting-derives-clauses) failed: 404 No error
[36](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:36)
- ./_site/blog/2017/05/31/first-dotty-milestone-release.html
[37](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:37)
  *  External link [dotty.epfl.ch/blog/2015/10/23/dotty-compiler-bootstraps.html](https://dotty.epfl.ch/blog/2015/10/23/dotty-compiler-bootstraps.html) failed: 404 No error
[38](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:38)
  *  External link [dotty.epfl.ch/docs/reference/enums/adts.html](https://dotty.epfl.ch/docs/reference/enums/adts.html) failed: 404 No error
[39](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:39)
  *  External link [dotty.epfl.ch/docs/reference/enums/enums.html](https://dotty.epfl.ch/docs/reference/enums/enums.html) failed: [40](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:40)4 No error
40
  *  External link [dotty.epfl.ch/docs/reference/new-types/intersection-types.html](https://dotty.epfl.ch/docs/reference/new-types/intersection-types.html) failed: 404 No error
[41](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:41)
  *  External link [dotty.epfl.ch/docs/reference/new-types/union-types.html](https://dotty.epfl.ch/docs/reference/new-types/union-types.html) failed: 404 No error
[42](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:42)
  *  External link [dotty.epfl.ch/docs/reference/other-new-features/trait-parameters.html](https://dotty.epfl.ch/docs/reference/other-new-features/trait-parameters.html) failed: 404 No error
[43](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:43)
  *  External link [dotty.epfl.ch/docs/release-notes/0.1.2.html](https://dotty.epfl.ch/docs/release-notes/0.1.2.html) failed: 404 No error
[44](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:44)
- ./_site/blog/2017/10/16/fourth-dotty-milestone-release.html
[45](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:45)
  *  External link [dotty.epfl.ch/docs/usage/version-numbers.html](https://dotty.epfl.ch/docs/usage/version-numbers.html) failed: 404 No error
[46](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:46)
- ./_site/blog/2018/04/19/scala-3.html
[47](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:47)
  *  External link [dotty.epfl.ch/docs/contributing/getting-started.html](https://dotty.epfl.ch/docs/contributing/getting-started.html) failed: 404 No error
[48](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:48)
  *  External link [dotty.epfl.ch/docs/reference/overview.html](https://dotty.epfl.ch/docs/reference/overview.html) failed: 404 No error
[49](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:49)
- ./_site/blog/2018/04/30/in-a-nutshell.html
[50](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:50)
  *  External link [dotty.epfl.ch/docs/reference/contextual/by-name-context-parameters.html](https://dotty.epfl.ch/docs/reference/contextual/by-name-context-parameters.html) failed: 404 No error
[51](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:51)
  *  External link [dotty.epfl.ch/docs/reference/metaprogramming/macros.html](https://dotty.epfl.ch/docs/reference/metaprogramming/macros.html) failed: 404 No error
[52](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:52)
  *  External link [dotty.epfl.ch/docs/reference/new-types/type-lambdas.html](https://dotty.epfl.ch/docs/reference/new-types/type-lambdas.html) failed: 404 No error
[53](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:53)
- ./_site/blog/2018/10/04/scaladoc-tables.html
[54](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:54)
  *  External link [dotty.epfl.ch/docs/usage/dottydoc.html](https://dotty.epfl.ch/docs/usage/dottydoc.html) failed: 404 No error
[55](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:55)
- ./_site/blog/2020/11/19/scala-3-forward-compat.html
[56](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:56)
  *  External link [dotty.epfl.ch/docs/reference/new-types/match-types.html](https://dotty.epfl.ch/docs/reference/new-types/match-types.html) failed: 404 No error
[57](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:57)
- ./_site/blog/2021/07/21/scala-3.0.2RC1-is-here.html
[58](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:58)
  *  External link [dotty.epfl.ch/blog/2021/06/07/scala3.0.1-rc1-release.html](https://dotty.epfl.ch/blog/2021/06/07/scala3.0.1-rc1-release.html) failed: 404 No error
[59](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:59)
  *  External link [dotty.epfl.ch/blog/2021/06/25/scala301-rc2.html](https://dotty.epfl.ch/blog/2021/06/25/scala301-rc2.html) failed: 404 No error
[60](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:60)
- ./_site/blog/2021/09/07/scala-3.0.2-released.html
[61](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:61)
  *  External link [dotty.epfl.ch/docs/reference/changed-features/structural-types-spec.html#limitations-of-structural-types](https://dotty.epfl.ch/docs/reference/changed-features/structural-types-spec.html#limitations-of-structural-types) failed: 404 No error
[62](https://github.com/scala/scala-lang/runs/5193204739?check_suite_focus=true#step:6:62)
htmlproofer 3.10.2 | Error:  HTML-Proofer found 39 failures!
```

still one error at https://github.com/scala/scala-lang/runs/5400201887 :

```
- ./_site/blog/2017/09/07/third-dotty-milestone-release.html
[11](https://github.com/scala/scala-lang/runs/5400201887?check_suite_focus=true#step:6:11)

[12](https://github.com/scala/scala-lang/runs/5400201887?check_suite_focus=true#step:6:12)
htmlproofer 3.10.2 | Error:  HTML-Proofer found 1 failure!
[13](https://github.com/scala/scala-lang/runs/5400201887?check_suite_focus=true#step:6:13)
  *  External link [dotty.epfl.ch/docs/usage/version-numbers.html](https://dotty.epfl.ch/docs/usage/version-numbers.html) failed: 404 No error
[14](https://github.com/scala/scala-lang/runs/5400201887?check_suite_focus=true#step:6:14)
Error: Process completed with exit code 1.
```

--- a/blog/_posts/2017-09-07-third-dotty-milestone-release.md
+++ b/blog/_posts/2017-09-07-third-dotty-milestone-release.md
@@ -18,7 +18,7 @@ You can learn more about Dotty on our [website](https://dotty.epfl.ch).
 
 <!--more-->
 
-This is our third scheduled release according to our [6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+This is our third scheduled release according to our 6-week release schedule.
 The [previous technology preview](/blog/2017/07/12/second-dotty-milestone-release.html) improved
 stability and reliability:
  - substantial improvement of quality of generated code for pattern matching

